README.md

```
@@ -77,12 +108,14 @@ The multicloud-gitops pattern is a set of best practices and tools that help org
 7. **Continuous Integration and Delivery**: Automated testing and deployment of code changes to production environments.
 ```
 
+### 2. **Available Patterns Query**
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)